include/boost/gil.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright 2005-2007 Adobe Systems Incorporated
+// Copyright 2018 Stefan Seefeld
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef boost_gil_hpp_
+#define boost_gil_hpp_
+
+#include <boost/gil/gil_config.hpp>
```

Comment:
`#include <boost/gil/version.hpp>` is missing

Reply:
Added, thanks.